packages/@react-spectrum/searchfield/docs/SearchField.mdx

```diff
@@ -0,0 +1,174 @@
+<SearchField label="Street address" necessityIndicator="label" />
+```
+
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the SearchField for 
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using 
+the `id` of the labeling element instead.
+
+```tsx example
+<label id="label-1">Name</label>
+<SearchField aria-labelledby="label-1" marginStart="10px" />
+```
+
+### Internationalization
+
+In order to internationalize a SearchField, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+```tsx example
+<SearchField label="Street address" isRequired necessityIndicator="label" marginEnd="20px" />
+<SearchField label="Street address" necessityIndicator="label" />
+```
+
+## Events
+
+The most commonly used handlers for events in SearchField are the:
+- `onChange` prop which is triggered whenever the value is edited by the user. 
+- `onSubmit` prop which is triggered whenever the value is submitted by the user. 
+- `onClear` prop which is triggered whenever the value is cleared by the user. 
+
+The example below uses `onChange`, `onSubmit`, and `onClear` to update a separate elements with the text entered into the SearchField.
+```tsx example
+function Example() {
+  let [currentText, setCurrentText] = React.useState();
+  let [submittedText, setSubmittedText] = React.useState();
+
+  return (
+    <div>
+      <SearchField
+        placeholder="Enter some text"
+        onClear={() => setCurrentText('')}
+        onChange={setCurrentText}
+        onSubmit={setSubmittedText}
+        label="Mirror"
+        marginEnd="20px"
+        value={currentText}
+      />
+      <div>Mirrored text: {currentText}</div>
+      <div>Submitted text: {submittedText}</div>
+    </div>
+  );
+}
+```
+
+## Props
+
+<PropTable component={docs.exports.SearchField} links={docs.links} />
+
+## Visual Options
+
+### Quiet 
+
+```tsx example
+<SearchField label="Email" placeholder="Email Address" isQuiet />
+```
+
+### Disabled
+
+```tsx example
+<SearchField label="Email" placeholder="Email Address" isDisabled />
+```
+
+### Read Only
+
+The `isReadOnly` boolean prop makes the SearchField's text content immutable. Unlike `isDisabled` the SearchField remains focusable 
+and the contents can still be copied. See [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) for more information.
+
+```tsx example
+<SearchField label="Email" defaultValue="abc@adobe.com" isReadOnly />
+```
+
+### Label Alignment
+
+Labels can be aligned to the "start" or the "end" of the SearchField. For left-to-right (LTR) languages, "start" refers to the left most edge
+of the SearchField and "end" refers to the right most edge. For right-to-left (RTL) languages, this is flipped.
```

Comment:
`this is flipped.` => `this is mirrored.`
I think in other docs we say mirrored, right?

Reply:
i think we use both (i copied this from TextField). I'll grep and adjust.